events/class_tests/create_events.py

```diff
@@ -3,54 +3,57 @@
-            "description": DESCRIPTION,
-            "date_time_start": DATE_TIME_START,
-            "date_time_end": DATE_TIME_END,
-            "color": COLOR,
-        }
+        "title": TITLE,
+        "location": LOCATION,
+        "description": DESCRIPTION,
+        "date_time_start": EVENT_DATE_TIME_START,
+        "date_time_end": EVENT_DATE_TIME_END,
+        "color": COLOR,
+    }
 
 
 @pytest.mark.django_db
 class TestCreateEventForm:
 
     @pytest.mark.parametrize("invalid_data", [
-            # title cannot be None
-            {'title': None, 'date_time_start': DATE_TIME_START, 'date_time_end': DATE_TIME_END, 'color': COLOR},
-            # title cannot be blank
-            {'title': "", 'date_time_start': DATE_TIME_START, 'date_time_end': DATE_TIME_END, 'color': COLOR},
-            # date_time_start cannot be None
-            {'title': TITLE, 'date_time_start': None, 'date_time_end': DATE_TIME_END, 'color': COLOR},
-            # date_time_end cannot be None
-            {'title': TITLE, 'date_time_start': DATE_TIME_START, 'date_time_end': None, 'color': COLOR},
-            # date_time_end must be bigger than date_time_start
-            {'title': TITLE, 'date_time_start': DATE_TIME_END, 'date_time_end': DATE_TIME_START, 'color': COLOR},
-            # date_time_end cannot be equal to date_time_start
-            {'title': TITLE, 'date_time_start': DATE_TIME_START, 'date_time_end': DATE_TIME_START, 'color': COLOR},
-        ], ids=[
-            "title is none",
-            "title is blank",
-            "star time is none",
-            "end time is none",
-            "end time is not bigger then start time",
-            "start time and end time are equal"
-        ]
-     )
+        # title cannot be None
+        {'title': None, 'date_time_start': EVENT_DATE_TIME_START, 'date_time_end': EVENT_DATE_TIME_END, 'color': COLOR},
+        # title cannot be blank
+        {'title': "", 'date_time_start': EVENT_DATE_TIME_START, 'date_time_end': EVENT_DATE_TIME_END, 'color': COLOR},
+        # date_time_start cannot be None
+        {'title': TITLE, 'date_time_start': None, 'date_time_end': EVENT_DATE_TIME_END, 'color': COLOR},
+        # date_time_end cannot be None
+        {'title': TITLE, 'date_time_start': EVENT_DATE_TIME_START, 'date_time_end': None, 'color': COLOR},
+        # date_time_end must be bigger than date_time_start
+        {'title': TITLE, 'date_time_start': EVENT_DATE_TIME_END,
+         'date_time_end': EVENT_DATE_TIME_START, 'color': COLOR},
+        # date_time_end cannot be equal to date_time_start
+        {'title': TITLE, 'date_time_start': EVENT_DATE_TIME_START,
+         'date_time_end': EVENT_DATE_TIME_START, 'color': COLOR},
+    ], ids=[
+        "title is none",
+        "title is blank",
+        "star time is none",
+        "end time is none",
+        "end time is not bigger then start time",
+        "start time and end time are equal"
+    ]
+                             )
```

Comment:
Can you change back to the indentation in the older file

Reply:
since the const name change from DATE_TIME_START to EVENT_DATE_TIME_START the line became long.

Reply:
Done, i guess